packages/analytics/dashboard-renderer/src/components/DashboardRenderer.cy.ts

```diff
@@ -495,7 +491,7 @@ describe('<DashboardRenderer />', () => {
     })
   })
 
-  it('picks 7 days and basic datasource if FF is enabled', () => {
+  it.skip('picks 7 days and basic datasource', () => {
```

Comment:
Why is this one skipped?

Reply:
No particular reason, other than it was failing before. Passing now :)